source/features/remove-unused-repo-tabs.tsx

```diff
@@ -0,0 +1,105 @@
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import fetchDom from '../helpers/fetch-dom';
+import getTabCount from '../github-helpers/get-tab-count';
+import looseParseInt from '../helpers/loose-parse-int';
+import {getWorkflows} from './next-scheduled-github-action';
+import abbreviateNumber from '../helpers/abbreviate-number';
+import {onlyShowInDropdown} from './more-dropdown';
+import {buildRepoURL, getRepo} from '../github-helpers';
+
+function tabCannotBeHidden(tab: HTMLElement | undefined): boolean {
```

Comment:
Let's _not_ use negative items. `true` here would mean `false`
```suggestion
function canTabBeHidden(tab: HTMLElement | undefined): boolean {
```

or
```suggestion
function mustKeepTab(tab: HTMLElement | undefined): boolean {
```